massgov/mayflower
# #DP-22483 Add topic section to org page.

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Updates to the section three up for supporting a compact version, see related issue.

## Related Issue / Ticket

- [DP-22483](https://jira.mass.gov/browse/DP-22483)

## Steps to Test


1. Checkout the flexible organization branch in your local openmass: https://github.com/massgov/openmass/pull/836
2. Checkout Mayflower to point to this branch.
3. Edit an organization and add a featured topics paragraph on the content tab of Organizations.
4. Make sure that both compact and normal versions looks well.

## Screenshots
<img width="900" alt="Captura de Pantalla 2021-07-23 a la(s) 17 26 38" src="https://user-images.githubusercontent.com/141685/126837940-6164b20b-f56f-45c3-ab7f-02af49d31f9d.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* Section Three Up

#### @TODO


*

#### Today I learned...
